### DISCUSSION TITLE
combined [Porous Flow] and [Geochemistry] modules

### DISCUSSION THREAD

**Discussion by Bearunner** [idaholab/moose#26114] [Q&A Modules: General]
Dear all, 

I want to simulate the chemical scaling process ([Geochemistry] module) within the wellbore (porostity=1 and initial Calcite = 0 in [Porous Flow] module). The input files are [here](https://github.com/Bearunner/combined-Porous-Flow-and-Geochemistry-modules).

The Species Concentration in injected water at 14°C are shown as follows.

Ca++ : 26.6 mg/L
Cl-:: 30.8 mg/L
SO4--:: 22.5 mg/L
HCO3-:12.2 mg/L
ph= 8.05 => H+ : 8.91E-09 mol/L

1st: I run the aquifer_equilibrium.i file to get the bulk composition of H+: 2.6136e-06 moles.

2nd: Insert the ulk composition of H+ into aquifer_geochemistry.i to run the combined  [Porous Flow] and [Geochemistry] modules.

But I get zero Calcite.

Could you please help me with this? Thanks.

B

**Comment by Bearunner** [comment]
Hello @cpgr  ,could you please help me with this? Thank you.

**Comment by josebastiase** [comment]
Hi, can you also share your json files?

**Reply by Bearunner** [reply]
hello. I use the defaulted json files of [Geochemistry] modules
